dev/com.ibm.ws.rest.handler.validator/src/com/ibm/ws/rest/handler/validator/internal/ValidatorRESTHandler.java

```diff
@@ -456,11 +471,19 @@ public final void handleRequest(RESTRequest request, RESTResponse response) thro
             return;
         }
 
+        //Throw 404 for /ibm/api/validation with no element name.
+        if ("/validation".equals(request.getPath()) ||
+            "/validation/".equals(request.getPath()) ||
+            "/validation//".equals(request.getPath())) {
```

Comment:
on this line we should do `startsWith("/validation//")` to account for people that do things like `/validation//myDs`. I could see this happen if they programmatically built the URL as `"/validation/" + element + "/" + otherStuff` and `element` was an empty string